src/triage/component/postmodeling/contrast/evaluator.py

```diff
@@ -0,0 +1,543 @@
+            merged_df = mat.merge(self.preds,
+                                  on='entity_id',
+                                  how='inner',
+                                  suffixes=('test', 'pred'))
+
+            self.pred_matrix = merged_df
+
+    def load_train_matrix(self, *path):
+        '''
+        Load training metrix (from s3 or system file). This object will be store 
+        as the train_matrix object of the class.
+
+        Arguments:
+            - Arguments inherited from _fecth_matrices:
+                - path: relative path to the triage matrices folder or s3 path
+        '''
+        if self.train_matrix is None:
+            self.train_matrix = self._fetch_matrix(self.train_matrix_uuid, *path)
+
+    def plot_score_label_distributions(self, 
+                                       save_file=False, 
+                                       name_file=None,
+                                       label_names = ('Label = 0', 'Label = 1'),
+                                       figsize=(16, 12),
+                                       fontsize=20):
+        '''
+        Generate a histogram showing the label distribution for predicted
+        entities:
+            - Arguments:
+                - save_file (bool): save file to disk as png. Default is False.
+                - name_file (string): specify name file for saved plot.
+                - label_names(tuple): define custom label names for class.
+                - figsize (tuple): specify size of plot. 
+                - fontsize (int): define custom fontsize. 20 is set by default.
+        '''
+        if self.preds is None:
+            self._load_predictions()
+
+        df_preds = self.preds.filter(items=['score', 'label_value'])
+        df_preds_0 = df_preds[df_preds.label_value == 0]
+        df_preds_1 = df_preds[df_preds.label_value == 1]
+
+        fig, ax = plt.subplots(1, figsize=figsize)
+        plt.hist(df_preds_0.score,
+                 bins=20,
+                 normed=True,
+                 alpha=0.5,
+                 color='skyblue',
+                 label=label_names[0])
+        plt.hist(list(df_preds_1.score),
+                 bins=20,
+                 normed=True,
+                 alpha=0.5,
+                 color='orange',
+                 label=label_names[1])
+        plt.axvline(df_preds_0.score.mean(),
+                    color='skyblue',
+                    linestyle='dashed')
+        plt.axvline(df_preds_1.score.mean(),
+                    color='orange',
+                    linestyle='dashed')
+
+        plt.legend(bbox_to_anchor=(0., 1.005, 1., .102),
+                   loc=8,
+                   ncol=2,
+                   borderaxespad=0.,
+                   fontsize=fontsize)
+        ax.set_xlabel('Frequency', fontsize=fontsize)
+        ax.set_ylabel('Score', fontsize=fontsize)
+        plt.title('Score Distribution across Predicted Labels')
+        plt.show()
+
+        if save_file:
```

Comment:
Probably is better to store them as `svg`

Reply:
Also, why is this the only plot function that stores the figure? What happen with the other plotting functions?